mk/meson/armv8.cfg

```diff
@@ -0,0 +1,8 @@
+[built-in]
+cpp_args = ['-fPIC']
+
+[host_machine]
+system = 'linux'
+cpu_family = 'aarch64'
+cpu = 'aarch64'
```

Comment:
for consistency this file could be renamed to aarch64.cfg (file name = cpu)

Reply:
good idea, done.